common.py

```diff
@@ -50,7 +49,7 @@ def build_command_list(command, ParamList, include_keys=True):
     return command
 
 
-def exec_command(context, command, shell=False, stdout_msg=None, cont_output=False):
+def exec_command(command, shell=False, stdout_msg=None, cont_output=False):
```

Comment:
This form is now in the gear-toolkit.
I don't know if you need the 'environ' for this.